packages/web-components/src/components/va-banner/va-banner.tsx

```diff
@@ -49,9 +49,9 @@ export class VaBanner {
 
   /* eslint-disable i18next/no-literal-string */
   /**
-   * The type of the banner. One of 'info', 'error', 'success', 'continue', or 'warning'. This affects both the icon of the AlertBox and the top border color.
+   * The type of the banner. This affects both the icon of the AlertBox and the top border color.
    * */
-  @Prop() type?: string = 'info';
+  @Prop() type?: "info" | "warning" | "error" | "success" | "continue" = 'info';
```

Comment:
I like this update. Storybook handles displaying these options really nice:
![Screenshot 2023-10-30 at 2 04 19 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/9d6bc3a9-7d36-46b0-97e0-a08d80e6dca0)